app/jobs/get_usps_proofing_results_job.rb

```diff
@@ -363,21 +372,24 @@ def handle_successful_status_update(enrollment, response)
       reason: 'Successful status update',
       job_name: self.class.name,
     )
-    enrollment.profile.activate_after_passing_in_person
     enrollment.update(
       status: :passed,
       proofed_at: proofed_at,
       status_check_completed_at: Time.zone.now,
     )
 
-    # send SMS and email
-    send_enrollment_status_sms_notification(enrollment: enrollment)
-    send_verified_email(enrollment.user, enrollment)
-    analytics(user: enrollment.user).idv_in_person_usps_proofing_results_job_email_initiated(
-      **email_analytics_attributes(enrollment),
-      email_type: 'Success',
-      job_name: self.class.name,
-    )
+    unless fraud_result_pending?(enrollment)
```

Comment:
Do we need to check the config setting here too? Based on your specs I think no, but I'm not totally clear on when `fraud_pending_reason` is available on the enrollment profile.

Reply:
Oh! I think we should.

We're not the first ones to use `fraud_pending_reason`, so the feature flag should toggle whether or not we act on it here. Seems like I need to clean up the test as well.

Reply:
Better yet, your suggestion in https://github.com/18F/identity-idp/pull/9928/files#r1467975629 gets us this for free.